code/datums/wounds/bones.dm

```diff
@@ -0,0 +1,388 @@
+	treat_text = "Immediate binding of affected limb, followed by surgical intervention ASAP."
+	examine_desc = "has a cracked bone sticking out of it"
+	occur_text = "cracks apart, exposing broken bones to open air"
+	severity = WOUND_SEVERITY_CRITICAL
+	interaction_efficiency_penalty = 4
+	limp_slowdown = 9
+	sound_effect = 'sound/effects/crack2.ogg'
+	threshold_minimum = 115
+	threshold_penalty = 50
+	disabling = TRUE
+	treatable_by = list(/obj/item/stack/sticky_tape/surgical, /obj/item/stack/medical/gauze, /obj/item/stack/medical/bone_gel)
+	status_effect_type = /datum/status_effect/wound/bone/critical
+	treat_priority = TRUE
+	scarring_descriptions = list("a section of janky skin lines and badly healed scars", "a large patch of uneven skin tone", "a cluster of calluses")
+
+/datum/wound/brute/bone/proc/gel(obj/item/stack/medical/bone_gel/I, mob/user)
+	if(gelled)
+		to_chat(user, "<span class='warning'>[user == victim ? "Your" : "[victim]'s"] [limb.name] is already coated with bone gel!</span>")
+		return
+
+	user.visible_message("<span class='danger'>[user] begins applying [I] to [victim]'s' [limb.name]...</span>", "<span class='warning'>You begin applying [I] to [user == victim ? "your" : "[victim]'s"] [limb.name]...</span>")
+
+	if(!do_after(user, base_treat_time * (user == victim ? 1.5 : 1), target = victim, extra_checks=CALLBACK(src, .proc/still_exists)))
+		return
+
+	I.use(1)
+	victim.emote("scream")
+	if(user != victim)
+		user.visible_message("<span class='notice'>[user] finishes applying [I] to [victim]'s [limb.name], emitting a fizzing noise!</span>", "<span class='notice'>You finish applying [I] to [victim]'s [limb.name]!</span>", ignored_mobs=victim)
+		to_chat(victim, "<span class='userdanger'>[user] finishes applying [I] to your [limb.name], and you can feel the bones exploding with pain as they begin melting and reforming!</span>")
+	else
+		var/painkiller_bonus = 0
+		if(victim.drunkenness)
+			painkiller_bonus += 5
+		if(victim.reagents && victim.reagents.has_reagent(/datum/reagent/medicine/morphine))
+			painkiller_bonus += 5
+		if(victim.reagents && victim.reagents.has_reagent(/datum/reagent/determination))
+			painkiller_bonus += 5
+
+		if(prob(25 + (20 * severity - 2) - painkiller_bonus)) // 25%/45% chance to fail self-applying with severe and critical wounds, -5 for each of the above painkillers
+			victim.visible_message("<span class='danger'>[victim] fails to finish applying [I] to [victim.p_their()] [limb.name], passing out from the pain!</span>", "<span class='notice'>You black out from the pain of applying [I] to your [limb.name] before you can finish!</span>")
+			victim.AdjustUnconscious(5 SECONDS)
+			return
+		victim.visible_message("<span class='notice'>[victim] finishes applying [I] to [victim.p_their()] [limb.name], grimacing from the pain!</span>", "<span class='notice'>You finish applying [I] to your [limb.name], and your bones explode in pain!</span>")
+
+	limb.receive_damage(100, wound_bonus=CANT_WOUND)
```

Comment:
er, you do know that this wound can form on your chest, right?

I can't WAIT for someone to accidentally crit themselves because they dealt 100 damage to themselves after SUCCESSFULLY treating their own chest with bone gel. That will be the PEAK of FUN AND INTUITIVE gameplay.

Reply:
oh good point, I'll fix that. Please turn down the snideness though, you can point out things I overlooked without all the theatrics.